rosdep/base.yaml

```diff
@@ -3138,7 +3141,7 @@ nvidia-cg:
 nvidia-cuda:
   debian: [nvidia-cuda-toolkit]
   gentoo: [nvidia-cuda-toolkit]
-  ubuntu: [nvidia-cuda-toolkit]
+  ubuntu: [nvidia-cuda-toolkit, g++-4.8]
```

Comment:
g++ 4.8 is certainly not the right choice across all Ubuntu distributions. Can you please reference some docs why you think this is necessary.